kyaml/kio/filters/fmtr_test.go

```diff
@@ -416,9 +416,9 @@ spec:
         image: nginx:1.7.9
         ports:
         - containerPort: 80
+      # this is a container
```

Comment:
This one looks like a bug fix to me. Does this mean we can remove https://github.com/kubernetes-sigs/kustomize/blob/master/kyaml/yaml/serialization.go#L8-L46?

Reply:
yes, tested it and it looks like we can safely remove it